### DISCUSSION TITLE
Open default application after login

### DISCUSSION THREAD

**Discussion by QBANIN** [goauthentik/authentik#11549] [Q&A]
I'm looking for a solution to open selected application/slug automatically after user login from my auth.example.com url. Every user (and there are many) has different application binded.

**Comment by joshuajeschek** [comment]
Did you manage to find a solution for this? I would really like to redirect different users to different default applications.

**Reply by dewi-tik** [reply]
You should be able to do this with a [redirect stage](https://docs.goauthentik.io/add-secure-apps/flows-stages/stages/redirect/). Add that stage to the end of your authentication flow and create a policy that dynamically sets the redirection target to whichever application launch URL you want. 

The policy could work based on an attribute that you set on the user, group membership etc, email address/username etc.
